Allow seperate requests for Camera & Photos perms

DanielDent opened the pull request:
In some use cases, permission for the camera is all that's needed. This change allows people to request either permission individually.

DanielDent committed:
Allow seperate requests for Camera & Photos perms

DanielDent commented:
@cla-bot check

tgpetrov commented:
Hi @DanielDent 
Thank you for your contribution.
Could you also mention the newly exposed methods in the README.md file?

DanielDent committed:
Document split permission request functions

The requestPermissions function remains available for requesting camera and saved photos access with a single function call, while requestCameraPermissions and
requestPhotosPermissions have been added to allow each permission to be requested on its own.

DimitarTodorov commented:
Hi @DanielDent 
Since the PR is made from fork, the CI checks would not pass. I've created the same [PR](https://github.com/NativeScript/nativescript-camera/pull/177) in our repo in order to run the CI checks in travis. Once the tests pass, we will merge the PR. 

Thanks again for the contribution,
NativeScript Team
